scripts/check-devices.py

```diff
@@ -41,7 +41,7 @@
 logger = logging.getLogger(__name__)
 
 
-def setup_logging(level: str = "info") -> None:
+def setup_logging(level: str) -> None:
```

Comment:
Need to change the Args now to `level (str)` and maybe mention tht the default is actually `WARNING` for some logger and INFO for the main one ;)

Reply:
done